valhalla/thor/attributes_controller.h

```diff
@@ -73,7 +73,7 @@ const std::string kEdgeTruckRoute = "edge.truck_route";
 const std::string kEdgeDefaultSpeed = "edge.default_speed";
 const std::string kEdgeDestinationOnly = "edge.destination_only";
 const std::string kEdgeIsUrban = "edge.is_urban";
-const std::string kEdgeTaggedNames = "edge.tagged_names";
+const std::string kEdgeTaggedValues = "edge.tagged_values";
```

Comment:
sadly this is a breaking api change. at best we need to support both names for this concept, ie keep both of these strings and if either is found then we turn on flag for this attribute

Reply:
Could you tell how does it actually work? I couldn't find usages of this const/value and just renamed for consistency.

Reply:
found it https://github.com/valhalla/valhalla/pull/3261/files#diff-fd61a9dafa6119b300d3fcc49f59456388c00d6851fb8cc325141be44daf58c4R819

Reply:
yeah sorry, that string there is part of the public json request api, you have to continue to support in the request when someone says they want "edge.tagged_names". the variable name doesnt matter but the string matchign in request parsing does. you should keep your new string as well and we'll just treat the old string as deprecated but supported. basically this needs to still support the old string as well as the new one: https://github.com/valhalla/valhalla/blob/master/src/worker.cc#L888-L895

Reply:
done it this way:
https://github.com/valhalla/valhalla/pull/3261/files#diff-dda1eacb6be34d7946468f09844f48890c1266509f634990ec27263271bd6f48R893-R899

wdyt?

Reply:
https://github.com/valhalla/valhalla/blob/79b7c59521817c32b1012af5886a691cbdf1736e/src/worker.cc#L894-L899

Reply:
@SiarheiFedartsou why are we renaming this.  @dgearhart are you ok with this?

Reply:
@gknisely I am assuming it is because z-level is not a name - I am okay with the more generic `tagged_values`

Reply:
the original idea was always supposed to be tags and values so that we could put any data in there we liked. this is more true to the original idea imho